core/camel-health/src/main/java/org/apache/camel/impl/health/RouteHealthCheck.java

```diff
@@ -54,7 +54,8 @@ protected void doCall(HealthCheckResultBuilder builder, Map<String, Object> opti
             if (route.getRouteController() != null || route.isAutoStartup()) {
                 if (status.isStarted()) {
                     builder.up();
-                } else if (status.isStopped()) {
+                }
+                if (status.isStopped()) {
```

Comment:
I's been long time since I've worked on this but I think the `else` should be preserved as if `status.isStarted()` returns true, there's no point to check also `status.isStopped()`

Reply:
Okay, will revert that to previous state. Thanks @lburgazzoli :)

Reply:
I don't think it effectively changes the result in this case but an if else makes it clear that it is either started or stopped whereas a list of is could be confusing like can the status be `started` and `stopped` at the same time ?

Reply:
Yes, i agree. Thank for clarifying. :)